github/github_test.go

```diff
@@ -186,6 +186,34 @@ func TestNewClient(t *testing.T) {
 	}
 }
 
+func TestNewEnterpriseClient(t *testing.T) {
+	baseURL := "https://custom-url/"
+	uploadURL := "https://custom-upload-url/"
+	c, _ := NewEnterpriseClient(baseURL, uploadURL, nil)
```

Comment:
I think we should check for errors in tests. We expect it to be `nil`, let's ensure that's the case with:

```Go
c, err := NewEnterpriseClient(baseURL, uploadURL, nil)
if err != nil {
	t.Fatalf("NewEnterpriseClient returned unexpected error: %v", err)
}
```

Otherwise, it might not be `nil`, and tests wouldn't catch that.